src/eko/anomalous_dimensions/as1.py

```diff
@@ -140,3 +140,48 @@ def gamma_singlet(N, s1, nf):
         [[gamma_qq, gamma_qg(N, nf)], [gamma_gq(N), gamma_gg(N, s1, nf)]], np.complex_
     )
     return gamma_S_0
+
+
+@nb.njit(cache=True)
+def gamma_QEDsinglet(N, s1, nf):
```

Comment:
I'm not sure this is the best name ever, but as @felixhekhorn keeps saying, "we're not good at naming".

In any case, let's brainstorm a bit about it :)

Reply:
Yes I agree. I didn't think much about it but we can always change it as we find a good name